webpack.config.js

```diff
@@ -38,11 +38,17 @@ module.exports = {
       template: path.resolve('./', 'index.html'),
       filename: 'index.html',
       hash: true
-    })
+    }),
+    new webpack.EnvironmentPlugin(
```

Comment:
Make sure webpack is aware of the env variables we care about so they are passed on to the app.